htdocs/takepos/css/split.css.php

```diff
@@ -0,0 +1,116 @@
+<?php
```

Comment:
Why adding a new css file just for this. The less css file we are the best it is. Can't you just merge the content into a section into an exsiting one, for example pos.css.php